src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -485,6 +485,8 @@
     <Compile Include="RoslynCodeTaskFactory\RoslynCodeTaskFactoryCodeType.cs" />
     <Compile Include="RoslynCodeTaskFactory\RoslynCodeTaskFactoryCompilers.cs" />
     <Compile Include="RoslynCodeTaskFactory\RoslynCodeTaskFactoryTaskInfo.cs" />
+    <Compile Include="System.Design.cs" />
+    <Compile Include="system.design\stronglytypedresourcebuilder.cs" />
```

Comment:
Intentional this is all lower case?

Reply:
I see it is being moved from somewhere else.

Reply:
Somebody's intention from the dark ages. That's the casing it's checked in under.